app/models/client.rb

```diff
@@ -123,6 +123,17 @@ def user_url
     ENV["VOLPINO_URL"] + "/users?client-id=" + symbol.downcase
   end
 
+  def self.push_to_index
+    self.find_each do |client|      
+      params = { "data" => { "type" => "clients", "attributes" => client.attributes } }
+      params["data"]["attributes"]["contact-email"]= params["data"]["attributes"]["contact_email"]
```

Comment:
This can be done simpler, along the lines of https://github.com/datacite/volpino/blob/master/app/controllers/application_controller.rb#L18-L20.